specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-09-01-preview/QueryPackQueries_API.json

```diff
@@ -0,0 +1,464 @@
+        "x-ms-examples": {
+          "QueryDelete": {
+            "$ref": "./examples/QueryPackQueriesDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorResponse": {
+      "description": "Describe the format of an Error response.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "QueriesResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "properties": {
+          "$ref": "#/definitions/LogAnalyticsQueryPackQueryProperties"
+        }
+      },
+      "description": "An Azure resource QueryPack-Query object"
+    },
+    "LogAnalyticsQueryPackQuery": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties that define an Log Analytics QueryPack-Query resource.",
+          "$ref": "#/definitions/LogAnalyticsQueryPackQueryProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/QueriesResource"
+        }
+      ],
+      "description": "A Log Analytics QueryPack-Query definition."
+    },
+    "LogAnalyticsQueryPackQueryProperties": {
+      "description": "Properties that define an Log Analytics QueryPack-Query resource.",
+      "properties": {
+        "queryId": {
+          "type": "string",
+          "description": "The unique ID of your application. This field cannot be changed.",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Unique name for your Query Pack."
+        },
+        "timeCreated": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Creation Date for the Log Analytics QueryPack, in ISO 8601 format.",
+          "format": "date-time"
+        },
+        "timeModified": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Last modified date of the Log Analytics QueryPack, in ISO 8601 format.",
+          "format": "date-time"
+        },
+        "author": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Object Id of user creating the query."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Azure tenant id."
+        },
+        "body": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Subscription id."
+        },
+        "linkedResourceId": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Resource id associated with the query."
```

Comment:
>Resource id associated with the query. [](start = 26, length = 38)

What is this linked resource used for?  How is it related to the query?

Reply:
thanks